### ISSUE TITLE
--output-ppm-stream does not obey -s

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nzinfo/gource#105]
```
Using the following command line:

gource -r 30 -1280x720 -s .5 --disable-progress --stop-at-end --highlight-users 
--output-ppm-stream - input.gource | ffmpeg -y -f image2pipe -vcodec ppm -r 30 
-i - -vcodec libx264 -b 8000000 -r 30 -threads 4 -vpre slow output.mp4

The issue is that gource bogs down quite a bit as it gets into the second half 
of the project log. This is understandable as there are a large number of files 
and lots going on, so I'm not really expecting realtime performance.

However, it'd be great if gource would actually obey the -s switch and stick to 
creating half-second days at 30 fps in the PPM output (as per the above 
config). The PPM output seems to mirror what's going on in the gource window 
real-time, which is a slow-motion version of what I want. In other words, if 
someone's outputting PPM, it'd be reasonable to expect that they don't really 
care about actual rendering speed - so the PPM should only have 15 frames per 
second in my case, no matter what happens in the project log itself.

In other words, as per the above config, the PPM has 15 frames for every day 
near the beginning. (half-second days at 30 fps) However, near the end, I can 
count as many as 24 frames per day.

Using gource 0.28 on Ubuntu.
```

Original issue reported on code.google.com by `amar...@gmail.com` on 1 Dec 2010 at 6:36
